data.proto

```diff
@@ -52,6 +52,8 @@ message AgentConfig {
   int32 actor_index = 5; // Used to figure out if an agent is the current_player in the observation space
   string device = 6;
   uint32 threads_per_worker = 7;
+  string repo_id = 8;
+  string filename = 9;
 }
```

Comment:
I think it'd be cleaner with a specific message named something like `HFHubModel` that would be used here.

Something like

```proto
message HFHubModel {
    string repo_id = 1;
    string filename = 2;
}

message AgentConfig {
    / / ...
    HFHubModel hf_hub_model = 8;
 }
```